lti_consumer/static/css/student.css

```diff
@@ -0,0 +1,41 @@
+.xblock-student_view.xblock-student_view-lti_consumer h2.problem-header {
```

Comment:
@douglashall I don't think edx-platform helps you in any way with compiling the CSS, but you can always add the compilation to `setup.py` to make sure it gets compiled on installation.